bot/exts/evergreen/splitify.py

```diff
@@ -0,0 +1,208 @@
+import math
+import random
+from concurrent.futures import ThreadPoolExecutor
+from io import BytesIO
+
+import discord
+from PIL import Image
+from discord.ext import commands
+
+
+MAX_SQUARES = 193_600
+
+EXECUTOR = ThreadPoolExecutor(10)
+
+
+class Splitify(commands.Cog):
+    """Splitifies your avatar!"""
```

Comment:
Splitify is a vague term.
```suggestion
    """Turn your avatar into a mosaic!"""
```